good_first_issues/utils/__init__.py

```diff
@@ -122,6 +125,7 @@ def web_server(html_data):
 
         with socketserver.TCPServer(("localhost", PORT), Handler) as httpd:
             print("serving at port", PORT)
+            webbrowser.open(f"http://127.0.0.1:{PORT}/")
```

Comment:
We can simplify this by directly passing the `0` port on the context manager.

```python
        with socketserver.TCPServer(("localhost", 0), Handler) as httpd:
            port = httpd.server_address()[1]
            print("serving at port", port)
            webbrowser.open(f"http://127.0.0.1:{port}")
```

@xkshitizx